cmd/asset-syncer/utils.go

```diff
@@ -184,6 +179,7 @@ func chartsFromIndex(index *helmrepo.IndexFile, r *models.Repo) []models.Chart {
 		}
 		charts = append(charts, newChart(entry, r))
 	}
+	sort.Slice(charts, func(i, j int) bool { return charts[i].ID < charts[j].Name })
```

Comment:
I don't understand this sorting, shouldn't this be (if needed) `charts[i].Name < charts[j].Name`?

Reply:
`charts[i].ID < charts[j].ID` is what I'd meant, thanks. The tests were failing intermittently because the order was not deterministic. Using Name still won't be deterministic as you can have `bitnami/apache` and `aardvark/apache`, the order of which would still be indeterminate if it's on the name only, I think?